### PR TITLE
Put jdbc statement effects on an environmental variable.

### PR DESCRIPTION
**Pull request by markhannum**
Rivers pointed out an issue with the current scheme: our current jdbc clients will receive an error code when communicating with an R6 server.  Enabling this logic on an environmental variable seemed to be the most straightforward fix, as it allows our jepsen-testing and current clients to continue working.

### CODE REVIEW

**Comment by riverszhang89** [comment]
:+1: Thank you for fixing it Mark! Would you mind me pushing a few commits to your branch? I have a JUnit test for the statement level query effects that I'd like to check in.

**Comment by markhannum** [comment]
Sure, np- I'll review whatever you send. :)

File: cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java [hunk 1]
```diff
@@ -164,7 +165,16 @@ public Comdb2Handle duplicate() {
...
         super(new ProtobufProtocol(), null);
         sets = new ArrayList<String>();
-        sets.add("set queryeffects statement");
+
+        /* export CDB2JDBC_STATEMENT_QUERYEFFECTS   -> enable
+         * export CDB2JDBC_STATEMENT_QUERYEFFECTS=1 -> enable
+         * export CDB2JDBC_STATEMENT_QUERYEFFECTS=0 -> disable
+         */
+        String envvar = System.getenv("CDB2JDBC_STATEMENT_QUERYEFFECTS");
+        statement_caching = (envvar != null && !envvar.equals("0"));
```
Comment (markhannum) [inline cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java]: Logic is completely sane- I'm going to change this variable name to 'statement_effects', and then merge.

**Comment by markhannum** [comment]
TX RIVERS!

**Comment by riverszhang89** [comment]
TX MARK!

**Pull request merged**